examples/basic_mysql_connection/index.js

```diff
@@ -4,17 +4,28 @@ var mysql = require('mysql');
 var config = JSON.parse(fs.readFileSync("config-secret.json"))
 
 var connection = mysql.createConnection({
-	host: config.host,
+	 host: config.host,
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+  database: config.database
 });
 
 connection.connect();
+const sqlStatments=['SELECT COUNT(*) as "Total No of Todos" FROM todos',
+					'SELECT * FROM todos WHERE Done=1',
+					'SELECT * FROM todos WHERE Done=0',
+					'SELECT * FROM todos GROUP BY Due DESC',
```

Comment:
Yes, in this case, I meant most recently added to the table.

Most recently due doesn't really make sense, but it's a bit of a confusing language thing - you would say "due furthest in the future" to describe what your query does.